src/MoBi.Core/Services/SimulationConfigurationFactory.cs

```diff
@@ -30,8 +30,7 @@ public SimulationConfiguration Create()
             SimulationSettings = _cloneManager.Clone(_projectRetriever.Current.SimulationSettings)
          };
 
-         _calculationMethodRepository.All().Each(
-            cm => simulationConfiguration.AddCalculationMethod(_cloneManager.Clone(cm)));
+         _calculationMethodRepository.All().Each(simulationConfiguration.AddCalculationMethod);
```

Comment:
@Yuri05 that's just a continuation of the earlier PR where we don't want to clone calculation methods anymore.